spring-cloud-gcp-examples/spring-cloud-gcp-storage-resource-example/src/main/java/com/example/WebController.java

```diff
@@ -38,7 +38,7 @@
 @RestController
 public class WebController {
 
-	@Value("gs://REPLACE_WITH_YOUR_BUCKET/my-file.txt")
+	@Value("gs://cccz-test-7/")
```

Comment:
I guess the changes on this file were committed by accident? :laughing: 
Saying because of the real bucket and the commented code.

Reply:
Woops!